### PR TITLE
Update aspnetcore-8.0.md

### LINKED ISSUE #29893: Performance improvements to named pipes transport
[[Enter feedback here]](https://devblogs.microsoft.com/dotnet/asp-net-core-updates-in-dotnet-8-preview-2/#performance-improvements-to-named-pipes-transport)

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: f30dbb9e-d64d-a925-01ca-00d9d035e6c1
* Version Independent ID: f30dbb9e-d64d-a925-01ca-00d9d035e6c1
* Content: [What's new in ASP.NET Core 8.0](https://learn.microsoft.com/en-us/aspnet/core/release-notes/aspnetcore-8.0?view=aspnetcore-7.0)
* Content Source: [aspnetcore/release-notes/aspnetcore-8.0.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/release-notes/aspnetcore-8.0.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-releasenotes**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #29893

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/release-notes/aspnetcore-8.0.md](https://github.com/dotnet/AspNetCore.Docs/blob/c9a8a01e821f2b972d8496a7ffbcdb207f27ef3b/aspnetcore/release-notes/aspnetcore-8.0.md) | [What's new in ASP.NET Core 8.0](https://review.learn.microsoft.com/en-us/aspnet/core/release-notes/aspnetcore-8.0?branch=pr-en-us-29894) |

### CODE REVIEW

File: aspnetcore/release-notes/aspnetcore-8.0.md [hunk 1]
```diff
@@ -294,6 +294,16 @@ builder.WebHost.ConfigureKestrel(serverOptions =>
...
 For more information about this feature and how to use .NET and gRPC to create an IPC server and client, see <xref:grpc/interprocess>.
 
+### Performance improvements to named pipes transport
+
+We’ve improved named pipe connection performance. Kestrel’s named pipe transport now accepts connections in parallel, and reuses <xref:System.IO.Pipes.NamedPipeServerStream> instances.
+
+Time to create 100,000 connections:
+
+* Before : 5.916 seconds
+* After  : 2.374 seconds
```
Comment (Rick-Anderson) [inline aspnetcore/release-notes/aspnetcore-8.0.md:304]:
```suggestion
* After &  : 2.374 seconds
```

File: aspnetcore/release-notes/aspnetcore-8.0.md [hunk 2]
```diff
@@ -294,6 +294,16 @@ builder.WebHost.ConfigureKestrel(serverOptions =>
...
 For more information about this feature and how to use .NET and gRPC to create an IPC server and client, see <xref:grpc/interprocess>.
 
+### Performance improvements to named pipes transport
+
+We’ve improved named pipe connection performance. Kestrel’s named pipe transport now accepts connections in parallel, and reuses <xref:System.IO.Pipes.NamedPipeServerStream> instances.
+
+Time to create 100,000 connections:
+
+* Before : 5.916 seconds
+* After &  : 2.374 seconds
```
Comment (Rick-Anderson) [inline aspnetcore/release-notes/aspnetcore-8.0.md:304]:
```suggestion
* After     : 2.374 seconds
```

File: aspnetcore/release-notes/aspnetcore-8.0.md [hunk 3]
```diff
@@ -294,6 +294,16 @@ builder.WebHost.ConfigureKestrel(serverOptions =>
...
 For more information about this feature and how to use .NET and gRPC to create an IPC server and client, see <xref:grpc/interprocess>.
 
+### Performance improvements to named pipes transport
+
+We’ve improved named pipe connection performance. Kestrel’s named pipe transport now accepts connections in parallel, and reuses <xref:System.IO.Pipes.NamedPipeServerStream> instances.
+
+Time to create 100,000 connections:
+
+* Before : 5.916 seconds
+* After     : 2.374 seconds
```
Comment (Rick-Anderson) [inline aspnetcore/release-notes/aspnetcore-8.0.md:304]: @guardrex  remind me how to get an extra space here so the numbers line up.

Comment (Rick-Anderson) [inline aspnetcore/release-notes/aspnetcore-8.0.md:304]:
```suggestion
* After &nbsp; : 2.374 seconds
```

Comment (tdykstra) [reply aspnetcore/release-notes/aspnetcore-8.0.md]: Did you try `&nbsp;`

Comment (Rick-Anderson) [reply aspnetcore/release-notes/aspnetcore-8.0.md]: > Did you try `&nbsp;`

yup, that works.

**Pull request merged**